index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Linkbrary</title>
+    <meta name="title" content="Linkbrary" />
+    <meta name="description" content="세상의 모든 정보를 쉽게 저장하고 관리해 보세요" />
+
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://linkbrary.codeit/" />
+    <meta property="og:title" content="Linkbrary" />
+    <meta property="og:description" content="세상의 모든 정보를 쉽게 저장하고 관리해 보세요" />
+    <meta property="og:image" content="https://metatags.io/images/meta-tags.png" />
+
+    <meta property="twitter:card" content="summary_large_image" />
+    <meta property="twitter:url" content="https://linkbrary.codeit/" />
+    <meta property="twitter:title" content="Linkbrary" />
+    <meta property="twitter:description" content="세상의 모든 정보를 쉽게 저장하고 관리해 보세요" />
+    <meta property="twitter:image" content="https://metatags.io/images/meta-tags.png" />
+    <link rel="stylesheet" href="styles/index_style.css" />
+    <link rel="stylesheet" href="styles/reset.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <div class="nav_fixed">
+        <a href="index.html">
+          <img src="./images/logo.svg" alt="홈으로 연결된 Linkbrary 로고" />
+        </a>
+        <a href="signin.html">
+          <span class="signin">로그인</span>
+        </a>
+        </div>
+      </nav>
+      <div class="title_header">
+      <div class="title">
+        <h1 class="slogan">
+          <span class="slogan_gradient background-clip-text">세상의 모든 정보</span>를<span class="space"></span>
+          쉽게 저장하고<br class="break"> 관리해 보세요
```

Comment:
media query로 br을 가려주는 것도 좋은 방법이지만 flex box로 감싸 자연스럽게 wrap 되도록 하면 break point를 신경쓰지 않아도 될듯합니다.
```suggestion
          <p style="display: flex; flex-wrap: wrap; justify-content: center;">
            <span style="display: inline-block;">쉽게 저장하고</span>
            <span style="display: inline-block;">관리해 보세요</span>
          </p>
```

가볍게 참고만해주세요!